experiment/pretest.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "question": "1. Which of the following gates can be used as even parity generator?",
+        "answers": {
+            "a": "XNOR",
+            "b": "XOR",
+            "c": "OR",
+            "d": "NOR"
+        },
+        "correctAnswer": "b"
+    },
+    {
+        "question": "2. Which of the following gates can be used as even parity generator?",
+        "answers": {
+            "a": "XNOR",
+            "b": "XOR",
+            "c": "OR",
+            "d": "NOR"
+        },
+        "correctAnswer": "a"
```

Comment:
We had not written this file so we didn't look into it in great detail earlier assuming that it would be correct. Q1 had the correct answer so removed Q2 and added a question in the end